tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
 envlist=
-    py{37,38,39,310}-core
-    py{37,38,39,310}-integration
-    py{37,38,39,310}-lint
+    py{37,38,39,310,311}-core
+    py{37,38,39,310,311}-integration
+    py{37,38,39,310,311}-lint
     docs
-    py{37,38,39,310}-wheel-cli
+    py{37,38,39,310,311}-wheel-cli
```

Comment:
We need to add `311` to line 51 in this file. As of now, it's not running the actual lint commands and so it will pass every time.

```ini
[testenv:py{37,38,39,310,311}-lint]
```